packages/catalog-server/src/lib/server/routes/bootstrap-packages.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Why `src/lib/server/routes` and not just `src/routes`?

Reply:
I dunno... my thinking was all the server parts in a folder with routes in a subfolder so that the files don't need a `-route` suffix.